libc/include/llvm-libc-macros/stdlib-macros.h

```diff
@@ -17,6 +17,11 @@
 #define EXIT_SUCCESS 0
 #define EXIT_FAILURE 1
 
+#ifndef MB_CUR_MAX
+// We only support the "C" locale right now, so this is a constant byte.
+#define MB_CUR_MAX 1
```

Comment:
is this going to be stable? I remember we ran into issues with `MB_LEN_MAX`. Also should this be moved to `limits-macros.h` to be in the same place as `MB_LEN_MAX`?

Reply:
So, in a real implementation this would be non-constant. And I don't think so, this is specifically mentioned in the standard as being provided by `stdlib.h`

```
       The <stdlib.h> header shall define the following macro which shall
       expand to a positive integer expression with type size_t:

       {MB_CUR_MAX}
                   Maximum number of bytes in a character specified by the
                   current locale (category LC_CTYPE).

       In the POSIX locale the value of {MB_CUR_MAX} shall be 1.
```